src/State/HubInterface.php

```diff
@@ -139,8 +140,9 @@ public function getIntegration(string $className): ?IntegrationInterface;
      * Sentry.
      *
      * @param TransactionContext $context Properties of the new transaction
+     * @param array<string, mixed> $customSamplingContext Additional context that will be passed to the SamplingContext
      */
-    public function startTransaction(TransactionContext $context): Transaction;
+    public function startTransaction(TransactionContext $context, ?array $customSamplingContext): Transaction;
```

Comment:
This is a huge breaking change, which is not acceptable without releasing a new major version. Rather than adding this argument for real, you can comment it and handle it with `func_get_arg` in the implementation class to avoid the break: we already did it in the past and it worked even though it was a bit messy